src/app/components/Arbitrage/Arbitrage.tsx

```diff
@@ -1,77 +1,120 @@
-import React, { useState, useEffect } from 'react';
-import { backendUrl, currentChainId } from '../../../utils/classifiers';
-import axios from 'axios';
-import { symbolByTokenAddress } from 'utils/blockchain/contract-helpers';
+import React, { useMemo } from 'react';
+import { bignumber } from 'mathjs';
 import { useTranslation } from 'react-i18next';
 import { translations } from 'locales/i18n';
-import { Popover, Icon } from '@blueprintjs/core';
+import { Icon, Popover } from '@blueprintjs/core';
+import { useSelector } from 'react-redux';
+import { assetByTokenAddress } from 'utils/blockchain/contract-helpers';
+import { backendUrl, currentChainId } from '../../../utils/classifiers';
+import { useFetch } from '../../hooks/useFetch';
+import { Asset } from '../../../types';
+import { selectWalletProvider } from '../../containers/WalletProvider/selectors';
+import { fixNumber } from '../../../utils/helpers';
+import { AssetSymbolRenderer } from '../AssetSymbolRenderer';
+import { toNumberFormat } from '../../../utils/display-text/format';
 
 const s = translations.swapTradeForm;
 
+const minUsdForOpportunity = 10;
+
+interface PoolData {
```

Comment:
I would probably create `model.ts` in the `Arbitrage` folder and move there this interface and the interface for `opportunities` so this file is less cluttered.